shell/platform/fuchsia/flutter/flatland_external_view_embedder.cc

```diff
@@ -336,6 +338,29 @@ void FlatlandExternalViewEmbedder::SubmitFrame(
       // Reset for the next pass:
       flatland_layer_index++;
     }
+
+    // TODO(fxbug.dev/104956): Setting hit regions for Flatland external view
+    // embedder should match with what is being done in GFX external view
+    // embedder.
+    // Set up the input interceptor at the top of the scene, if applicable.  It
+    // will capture all input, and any unwanted input will be reinjected into
+    // embedded views.
+    if (intercept_all_input_) {
+      fuchsia::ui::composition::TransformId input_interceptor_node =
```

Comment:
You need to add this to child_transforms_ so it can be cleaned up with the rest.

Reply:
Done